packages/components/src/Actions/ActionFile/__snapshots__/ActionFile.test.js.snap

```diff
@@ -38,6 +39,7 @@ exports[`ActionFile should display a Progress indicator if set 1`] = `
   <label
     className="btn theme-btn-file disabled"
     htmlFor={42}
+    role="button"
```

Comment:
label has role=button and input has sr-only class? something goes wrong here!

Reply:
This is du to the ActionFile component.
We want to hide the the input, and only display a sort of `button` to trigger it.

*Before*
* the input was hidden : not focusable
* the label has button shape but it was wrong, AT ignore it

*Now*
* the input is displayed, so focusable, but `sr-only` kind of hide it
* the label has button shape and we indicate that it serves as button. This enables the click on it to trigger the input. If you want a `<button>` instead, we would have to add a button, hide the input to not be focusable, and trigger the input via javascript and DOM manipulation on button click.

Does it sound good ?

Reply:
No, screen reader will use HTML as it:

```html
<form>
  <label for="mySelectedFile">File input label</label>
  <input type="file" id="mySelectedFile">

  <button type="submit">Send my file</button>
</form>
```

☝️ This is fully accessible!

With your solution, you made a label focusable, it's wrong, it's not a button.

What we want, it's:

```html
  <input
    className="theme-action-file-label"
    className="theme-action-file-input"
    disabled={false}
    id={42}
    name={undefined}
    onChange={[Function]}
    type="file"
  />
  <label
    className="btn theme-btn-file"
    htmlFor={42}
  >
    <Icon
      name="talend-caret-down"
      transform="rotate-180"
    />
    <span>
      Click me
    </span>
  </label>
```

With this kind of CSS:
```scss
input { 
  @extends sr-only; // yes, agreed
  
  &:focus + label { 
	box-shadow: // my focused btn outline
  }
}
```

Reply:
Ok understood. I remove the role. Thanks